Bugsnag/Client/BugsnagClient.m

```diff
@@ -617,6 +618,11 @@ - (void)start {
     // notification not received in time on initial startup, so trigger manually
     [self willEnterForeground:self];
     [self.pluginClient loadPlugins];
+
+    // add metadata about app/device
+    NSDictionary *systemInfo = [BSG_KSSystemInfo systemInfo];
+    [self.metadata addMetadata:BSGParseAppMetadata(@{@"system": systemInfo}) toSection:BSGKeyApp];
+    [self.metadata addMetadata:BSGParseDeviceMetadata(systemInfo) toSection:BSGKeyDevice];
```

Comment:
Seems to make sense to wrap this in a "system" key to match App and then you don't need to check in two places in `BSGParseDeviceMetadata `.

```suggestion
    [self.metadata addMetadata:BSGParseDeviceMetadata(@{@"system": systemInfo}) toSection:BSGKeyDevice];
```